src/App.css

```diff
@@ -14,7 +18,7 @@
   }
 }
 
-.App-header {
+.Dpp-header {
```

Comment:
This should be changed back to App.

Reply:
App header is used in the code so I removed it.